READMEs/parameters.md

```diff
@@ -7,45 +7,58 @@ SPDX-License-Identifier: Apache-2.0
 
 We can set any config parameter (a) via an envvar, or (b) via a config file. Envvar values override config file values.
 
-An `Ocean` instance will hold a `Config` instance that holds various config parameters. These parameters can be set in a variety of ways:
+An `Ocean` instance will hold a `Config` instance that holds various config parameters. These parameters need to get set. This is set based on what's input to `Ocean` constructor:
 
-1. Config object input
-    * Filled from ExampleConfig: `Ocean(ExampleConfig.get_config())` (recommended)
-    * Filled from config file: `Ocean(Config('config.ini'))`
-    * Filled from config file specified by `OCEAN_CONFIG_FILE` envvar: `Ocean(Config())`
-    * Filled from config dict: `Ocean(Config('network':..}))`
-2.  dict input: `Ocean({'network':..})` (deprecated because it doesn't support all config options)
+1.  dict input: `Ocean({'network':..})`
+2.  Config object input: `Ocean(Config('config.ini'))`
```

Comment:
For the second point `Config object input` we could let the unordered list which described each method of config creation to be clearer for the user. That's just my opinion.

Reply:
Thanks for the comment!  

All of the changes to parameters.md were reverted and moved over to #511 so I'll take the liberty of copying this comment over there for you.

Reply:
On second thought, the `Config object input` line doesn't exist in #511 so I'm unable to copy this for you.